openpower/configs/stonewither_defconfig

```diff
@@ -0,0 +1,67 @@
+BR2_powerpc64le=y
+BR2_powerpc_power8=y
+BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_OP_BUILD_PATH)/patches/stonewither-patches"
```

Comment:
Should this currently just point to witherspoon-patches? I can imagine the same issues we sometimes patch around on witherspoon would apply to stonewither?
(I'm thinking of our current stop4/stop5 disabling on witherspoon)